packages/yoga/src/Tag/native/Informative.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import styled, { withTheme } from 'styled-components';
+import { func, oneOfType, oneOf, bool, node } from 'prop-types';
+import { hexToRgb } from '@gympass/yoga-common';
+import { StyledTag, StyledText } from './Tag';
+
+const Informative = styled(StyledTag)`
+  ${({
+    variant,
+    theme: {
+      yoga: {
+        colors: {
+          text,
+          feedback: {
+            [variant]: { light = text.secondary },
```

Comment:
We have a pattern defined to this type of destruct? At some places we are using `[variant]: color = { light: text.secondary }` and others like this line, if we don't have a pattern defined we could just choose one and use it on all Tag components.